clusterloader2/testing/experimental/storage/pod-startup/config.yaml

```diff
@@ -7,12 +7,20 @@
   {{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
 
 # Test Variales
-  {{$PODS_PER_NODE := .PODS_PER_NODE}}
+  {{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 1}}
   {{$DEPLOYMENT_TEMPLATE_PATH := .DEPLOYMENT_TEMPLATE_PATH }}
-  {{$VOLUMES_PER_POD := .VOLUMES_PER_POD}}
+  {{$VOLUMES_PER_POD := DefaultParam .VOLUMES_PER_POD 1}}
   {{$VOLUME_TEMPLATE_PATH := .VOLUME_TEMPLATE_PATH}}
+  # Set this to false if metrics data is not needed.
+  {{$GATHER_METRICS := DefaultParam .GATHER_METRICS true}}
+  {{$START_PODS := DefaultParam .START_PODS true}}
   {{$PROVISION_VOLUME := DefaultParam .PROVISION_VOLUME false}}
-  {{$USE_CSI := DefaultParam .USE_CSI false}}
+  {{$STORAGE_CLASS := DefaultParam .STORAGE_CLASS "csi-gce-pd"}}
+  # Used for provisioning volumes via CSI. Without it,
+  # the PV controller must update each PVC before external-provisioner
+  # starts to work on it, which limits the volume creation rate.
+  # With it, volume creation starts immediately.
+  {{$PROVISIONER := DefaultParam .PROVISIONER ""}}
```

Comment:
What happens when this is set to empty string?

Reply:
Due to the if check, the annotation is not set and the volume controller must add it after the PVC got created.